hedera-mirror-rosetta/app/errors/errors.go

```diff
@@ -0,0 +1,45 @@
+package errors
+
+import "net/http"
+
+var Errors = map[string]Error{
+	BlockNotFound:        New(BlockNotFound, http.StatusBadRequest, true),
+	StartMustBeBeforeEnd: New(StartMustBeBeforeEnd, http.StatusBadRequest, true),
```

Comment:
imo the `retriable` should be set to `false` here.

From Rosetta Docs: "An error is retriable if the same request may succeed if submitted again."
In this case, it won't succeed if submitted again